--- origsrc/libapreq2-2.08/acinclude.m4	2006-08-08 23:26:45.000000000 -0500
+++ src/libapreq2-2.08/acinclude.m4	2008-06-16 12:39:12.359375000 -0500
@@ -172,7 +172,7 @@
         dnl Absolute source/build directory
         abs_srcdir=`(cd $srcdir && pwd)`
         abs_builddir=`pwd`
-        top_builddir="$abs_srcdir"
+        top_builddir="$abs_builddir"
 
         if test "$abs_builddir" != "$abs_srcdir"; then
           USE_VPATH=1
--- origsrc/libapreq2-2.08/build/doxygen.conf.in	2006-08-08 23:26:44.000000000 -0500
+++ src/libapreq2-2.08/build/doxygen.conf.in	2008-06-16 12:39:12.375000000 -0500
@@ -27,7 +27,7 @@
 WARN_IF_DOC_ERROR      = YES
 WARN_FORMAT            = "$file:$line: $text"
 
-INPUT                  = . include module/apache module/apache2
+INPUT                  = . include module/apache2
 FILE_PATTERNS          = *.h *.dox CHANGES STATUS
 RECURSIVE              = NO
 EXTRACT_STATIC         = YES
--- origsrc/libapreq2-2.08/glue/perl/Makefile.PL	2006-08-08 23:26:45.000000000 -0500
+++ src/libapreq2-2.08/glue/perl/Makefile.PL	2008-06-16 12:39:12.375000000 -0500
@@ -62,11 +62,11 @@
     $perl_lib =~ s{\\}{\\\\}g;
 }
 else {
-    slurp my $config => "$base_dir/config.status";
-    $config =~ /^s,\@APACHE2_INCLUDES\@,([^,]+)/m or
+    slurp my $config => "$base_dir/Makefile";
+    $config =~ /^APACHE2_INCLUDES = ([ \S]+)/m or
         die "Can't find apache include directory";
     $apache_includes = $1;
-    $config =~ /^s,\@APR_INCLUDES\@,([^,]+)/m or
+    $config =~ /^APR_INCLUDES = ([ \S]+)/m or
         die "Can't find apache include directory";
     $apache_includes .= " $1";
 
@@ -80,11 +80,11 @@
 
     $apr_libs ="";
 
-    $config =~ m/^s,\@APREQ_LIBNAME\@,([^,]+)/m or
+    $config =~ /^APREQ_LIBNAME = (\S+)/m or
         die "Can't find apreq libname";
     $apreq_libname = $1;
 
-    $config =~ m/^s,\@PACKAGE_VERSION\@,([^,]+)/m or
+    $config =~ /^PACKAGE_VERSION = (\S+)/m or
         die "Can't find package version";
     $version = $1;
 
--- origsrc/libapreq2-2.08/library/Makefile.am	2006-08-08 23:26:44.000000000 -0500
+++ src/libapreq2-2.08/library/Makefile.am	2008-06-16 12:39:12.390625000 -0500
@@ -5,7 +5,7 @@
 libapreq2_la_SOURCES = util.c version.c cookie.c param.c parser.c \
                        parser_urlencoded.c parser_header.c parser_multipart.c \
 	               module.c module_custom.c module_cgi.c error.c
-libapreq2_la_LDFLAGS = -version-info @APREQ_LIBTOOL_VERSION@ @APR_LTFLAGS@ @APR_LIBS@
+libapreq2_la_LDFLAGS = -version-info @APREQ_LIBTOOL_VERSION@ -no-undefined @APR_LTFLAGS@ @APR_LIBS@
 
 test: all
 	cd t; $(MAKE) test
